internal/logs/command.go

```diff
@@ -0,0 +1,44 @@
+package logs
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/brevdev/brev-go-cli/internal/brev_api"
+	"github.com/brevdev/brev-go-cli/internal/cmdcontext"
+	"github.com/brevdev/brev-go-cli/internal/terminal"
+)
+
+func NewCmdLogs(t *terminal.Terminal) *cobra.Command {
+	var task string
+	cmd := &cobra.Command{
+		Use:   "logs",
+		Short: "Tail logs",
+		Long: `Tail logs of current project Ex:
+
+		brev logs --task server
+		brev logs --task worker
+		brev logs --task packager
+	`,
```

Comment:
We _just now_ moved to using the Example field here
```suggestion
		Long: "Tail logs of current project."
		Example: `  brev logs --task server  
  brev logs --task worker
  brev logs --task packager`,
```

(note that the tick marks here will respect whitespace precisely as it is written, this includes newlines)